.claude/skills/orchestrate/scripts/recycle-agent.sh

```
@@ -10,10 +10,10 @@
 
 set -euo pipefail
 
-if [ $# -lt 3 ]; then
+if [ $# -lt 3 ]; 
```

Shellcheck (0.11.0)

[error] Couldn't parse this if expression. Fix to allow more checks.

(SC1073)